ansible/roles/common/tasks/os.yml

```diff
@@ -17,6 +17,12 @@
 
 - name: "set swappiness to zero"
   sysctl: name=vm.swappiness value=0
+- name: "set vm.min_free_kbytes to 1G"
```

Comment:
Were there problems you encountered that these settings helped resolve?

Reply:
Thanks for the question, Keith. In our performance tests on Azure, we found that with larger numbers of test clients (we are testing with cingest) per tserver node, and with large native maps, we were hitting strange "page allocation failure" errors (not OOM killer though) with symptoms similar to those described in these threads:
* https://bugzilla.kernel.org/show_bug.cgi?id=107111
* https://access.redhat.com/solutions/90883
* https://serverfault.com/questions/236170/page-allocation-failure-am-i-running-out-of-memory

Again, if you prefer we keep these conditional when cluster_type == 'azure', that's easy for us to do.